hera_cal/abscal.py

```diff
@@ -1238,6 +1254,13 @@ def abscal_run(data_file, model_files, calfits_infile=None, verbose=True, overwr
         gain_dict = merge_gains(gain_list)
         flag_dict = odict(map(lambda k: (k, np.zeros((Ntimes, Nfreqs), np.bool)), gain_dict.keys()))
 
+        # ensure reference antenna phase has been projected out (i.e. set to zero)
+        for p in AC.gain_pols:
+            refant_phs = gain_dict[(AC.refant, p)] / np.abs(gain_dict[(AC.refant, p)])
```

Comment:
I'd call this something other than `refant_phs`... it's more like a unit vector in the complex plane than a phase.

Reply:
i think `refant_phasor` is more appropriate